### PR TITLE
moving integration tests from ui to rawls

### PR DESCRIPTION
**Pull request by UrsaStutsman**
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check that the **Product Owner** has signed off on any user-facing changes
- [ ] **Submitter**: Make sure Swagger is updated if API changes
  - [ ] **...and Orchestration's Swagger too!**
- [ ] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Database checks:
  * If PR includes new or changed db queries, include the explain plans in the description
  * Make sure liquibase is updated if appropriate
  * If doing a migration, take a backup of the
  [dev](https://console.cloud.google.com/sql/instances/terraform-qfarbdq3lrexxck5htofjs5z6m/backups?project=broad-dsde-dev&organizationId=548622027621)
  and
  [alpha](https://console.cloud.google.com/sql/instances/terraform-r4caezzc35c4tb7pgdhwkmme4y/backups?project=broad-dsde-alpha&organizationId=548622027621)
  DBs in Google Cloud Console
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] Tell your tech lead (TL) that the PR exists if they want to look at it
- [ ] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via Slack and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Review by jmthibault79 — APPROVED** [review]
Does not include https://github.com/broadinstitute/firecloud-ui/pull/1124 but looks good for the test state before that.

Is there a current plan to turn on automatic PRs for the Rawls repo and update to the latest tests?

**Comment by UrsaStutsman** [comment]
@jmthibault79 I'm rebasing all branches with their dev counterparts today. Not entirely sure what you mean by "automatic PRs"... The plan is (at least for this first iteration), that each service with these tests will run on PR's the same way UI does currently. The swat pipeline will be updated so that if you're manually running it you can pick which services' swat you want to run.

**Comment by jmthibault79** [comment]
That is what I'm looking for.  Thanks!

**Review by dvoet — APPROVED** [review]

File: automation/docker/users.json.ctmpl [hunk 1]
```diff
@@ -77,6 +77,10 @@
     "hermione": "hermione.owner@test.firecloud.org",
     "ron": "ron.weasley@test.firecloud.org"
   }
```
Comment (jmthibault79) [inline automation/docker/users.json.ctmpl]: add comma

File: automation/docker/users.json.ctmpl [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  },
+  "temps": {
+    "luna": "luna.temp@test.firecloud.org",
+    "neville": "neville.temp@test.firecloud.org"
+  },
+  "notebookswhitelisted": {
+    "hermione": "hermione.owner@test.firecloud.org",
+    "ron": "ron.weasley@test.firecloud.org"
+  }
+  "campaignManagers": {
```
Comment (jmthibault79) [inline automation/docker/users.json.ctmpl]: also needed in `quality` section

**Pull request merged**